src/styles/todo.scss

```diff
@@ -0,0 +1,102 @@
+.todo {
+  position: relative;
+
+  display: grid;
+  grid-template-columns: 45px 1fr;
+  justify-items: stretch;
+
+  font-size: 24px;
+  line-height: 1.4em;
+  border-bottom: 1px solid #ededed;
+  
+  &:last-child {
+    border-bottom: 0;
+  }
+
+  &__status-label {
+    cursor: pointer;
+    background-image: url("data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20width%3D%2240%22%20height%3D%2240%22%20viewBox%3D%22-10%20-18%20100%20135%22%3E%3Ccircle%20cx%3D%2250%22%20cy%3D%2250%22%20r%3D%2250%22%20fill%3D%22none%22%20stroke%3D%22%23ededed%22%20stroke-width%3D%223%22/%3E%3C/svg%3E");
```

Comment:
Use local files and pass the path of the file to the background attr, it is a faster solution.